.golangci.yml

```diff
@@ -20,3 +20,6 @@ issues:
     - path: magefile\.go
       linters:
         - deadcode
+    - path: internal\/corazawaf\/rule\.go
+      linters:
+        - staticcheck
```

Comment:
Do we truly need this?

Reply:
reflect.StringHeader was deprecated in go 1.21,  this was causing the lint check to fail.  I wasn't sure how best to handle that other than adding an exception for now.

Reply:
We can also use inline linter nochecks

Reply:
I think the lint version is going to be related to the Go version. So once you downgrade back to `go 1.20`, The current linter (v1.54) should work back again

Reply:
I was able to remove this.  PR #1145 now includes a one liner to exclude the static check for the use of reflect.StringHeader